docs/install_linux.md

```diff
@@ -32,7 +32,7 @@ Upgrade:
 
 ```bash
 sudo apt update
-sudo apt install gh
+sudo apt upgrade gh 
```

Comment:
Is there a functional difference between `install` and `upgrade` when upgrading a package?

Reply:
yes, and `install` is what we want here. `upgrade` will upgrade all upgradable packages on the system even if a package argument is supplied.